src/json/mahler-symphony-video.js

```diff
@@ -78,17 +78,17 @@ export default {
                     {
                       id:
                         'https://dlib.indiana.edu/iiif_av/mahler-symphony-3/CD2/high/320Kbps.mp4',
-                      type: 'Audio',
-                      format: 'audio/mp4; codec..xxxxx',
+                      type: 'Video',
```

Comment:
This works for now, but maybe it would be good to get a real video example instead of using a contrived example of a video canvas with only audio on it.

Reply:
Yes, I agree :+1: